whisper.h

```diff
@@ -208,6 +209,7 @@ extern "C" {
         bool speed_up;  // speed-up the audio by 2x using Phase Vocoder
         int  audio_ctx; // overwrite the audio context size (0 = use default)
 
+        const std::vector<whisper_token> * prompt_tokens;
```

Comment:
The idea is correct, but the problem is we cannot afford to use C++ constructs like `std::vector` in the C-style API. If we use it, then it will break compatibility with other programming languages that want to use the library.

Instead of `std::vector<whisper_token>` you will have to pass `whisper_token * prompt_tokens` and `int prompt_n_tokens`. I know it's ugly, but it's portable.

Reply:
Tried to do it, no idea if this is what you mean. I hate it, I wish we could have the same performance with Go 😆